roles/postgresql/tasks/install.yml

```diff
@@ -3,6 +3,7 @@
     name:
       - postgresql
       - postgresql-client
+      - python3-psycopg2    # Moodle install fails without this
```

Comment:
To support Ansible modules: [postgresql_db](https://docs.ansible.com/ansible/latest/modules/postgresql_db_module.html) and [postgresql_dbuser](https://docs.ansible.com/ansible/latest/modules/postgresql_user_module.html) available from community.postgresql

Reply:
Correct spelling is `postgresql_user`

Thanks as this documentation really helps.